src/Stripe.net/Services/Topups/StripeTopupCreateOptions.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Stripe
+{
+    public class StripeTopupCreateOptions : StripeBaseOptions
+    {
+        /// <summary>
+        /// A positive integer in the smallest currency unit (e.g., 100 cents to top up $1.00 or 100 to top up ¥100, a 0-decimal currency) representing how much to top up your Stripe balance.
+        /// </summary>
+        [JsonProperty("amount")]
+        public int? Amount { get; set; }
+
+        /// <summary>
+        /// 3-letter ISO code for currency.
+        /// </summary>
+        [JsonProperty("currency")]
+        public string Currency { get; set; }
+
+        /// <summary>
+        /// An arbitrary string which you can attach to a Top-up object. It is displayed when in the web interface alongside the Top-up.
+        /// </summary>
+        [JsonProperty("description")]
+        public string Description { get; set; }
+
+        /// <summary>
+        /// A set of key/value pairs that you can attach to a Top-up object. It can be useful for storing additional information in a structured format.
+        /// </summary>
+        [JsonProperty("metadata")]
+        public Dictionary<string, string> Metadata { get; set; }
+
+        [JsonProperty("source")]
+        public string SourceId { get; set; }
```

Comment:
I named this `SourceId` because it has to be a real Source id `src_XXX` and can't be just a token. I'm ambivalent on whether `Source` is a better name but worry that one day we'll turn that into a "string or hash" version again. What do you think?

If so, we likely want to match that on the `StripeTopup` resource.

Reply:
`SourceId` works for me — there's probably some inconsistency throughout, but it matches well with the expandable conventions that we have in here (e.g., `BalanceTransationId`).